zoomus/client.py

```diff
@@ -32,6 +32,7 @@
         "report": components.report.ReportComponentV2,
         "user": components.user.UserComponentV2,
         "webinar": components.webinar.WebinarComponentV2,
+        "room": components.room.RoomComponentV2,
```

Comment:
This might be a bit nit-picky, but can I trouble you to put this in alphabetical order (i.e. after `report`)?